cars/car.js

```diff
@@ -1,9 +1,21 @@
 module.exports = Car
 
-function Car(mpg) {
-
+function Car({ gallons, odometer, trips }) {
+  if (!(this instanceof Car)) {
+    return new Car({ gallons, odometer, trips  });
+  }
+	this.gallons = 0
+	this.odometer = 0
+	this.trips = []
```

Comment:
The Car arguments shouldn't be changed - esp. as `gallons`, `odometer`, and `trips` vars aren't being used.
But `mpg` still needs to be set on the `this`.